aws/redshift/views/census_details.sql

```diff
@@ -9,8 +9,6 @@ with csa as
       when cso.school_code is null and ss.school_type in ('public', 'charter') then 0 -- only include known status for public and charter schools, as we don't have data for private schools
     end as teaches_csa
   from analysis.school_stats ss
-    left join dashboard_production.ap_school_codes sc on sc.school_id = ss.school_id
```

Comment:
Has some from @code-dot-org/red-team reviewed the changes to these Redshift-related queries?

Reply:
Thanks for calling this out @sureshc.  I need some time to review.